tests/e2e/default/users/add-user.wdio-spec.js

```diff
@@ -79,6 +80,40 @@ describe('User Test Cases ->', () => {
       await usersAdminPage.saveUser();
       expect(await usersAdminPage.getAllUsernames()).to.include.members([username]);
     });
+
+    it('should hide and reveal password value, and add user with a revealed password', async () => {
+      await usersAdminPage.inputAddUserFields(
+        username,
+        'Jack',
+        onlineUserRole,
+        districtHospital.name,
+        person.name,
+        password
+      );
+
+      let revealedPassword = await usersAdminPage.togglePassword();
+      expect(revealedPassword.type).to.equal('text');
+      expect(revealedPassword.value).to.equal(password);
+      expect(revealedPassword.confirmType).to.equal('text');
+      expect(revealedPassword.confirmValue).to.equal(password);
+
+      await usersAdminPage.setUserPassword(password2);
+      await usersAdminPage.setUserConfirmPassword(password2);
+      const hiddenPassword = await usersAdminPage.togglePassword();
+      expect(hiddenPassword.type).to.equal('password');
+      expect(hiddenPassword.value).to.equal(password2);
+      expect(hiddenPassword.confirmType).to.equal('password');
+      expect(hiddenPassword.confirmValue).to.equal(password2);
+
+      revealedPassword = await usersAdminPage.togglePassword();
+      expect(revealedPassword.type).to.equal('text');
+      expect(revealedPassword.value).to.equal(password2);
+      expect(revealedPassword.confirmType).to.equal('text');
+      expect(revealedPassword.confirmValue).to.equal(password2);
+
+      await usersAdminPage.saveUser();
+      expect(await usersAdminPage.getAllUsernames()).to.include.members([username]);
```

Comment:
```suggestion
      expect(await usersAdminPage.getAllUsernames()).to.include.members([username2]);
```

Reply:
We need a new username because usernames must be unique and the previous test already saved that initial username. 

I've made those slight changes run the test and it works. 

Thanks for adding the test 🥳 

How is it going with end to end test for update-password-component.html. ? 

Have you been able to run the e2e tests locally, or are you facing any challenge ?